### DISCUSSION TITLE
How can I get the version of the self-hosted runner from the shell?

### DISCUSSION THREAD

**Discussion by toast-gear** [community/community#160742] [Actions] [answered]
I know there is a flag you can provide either the `config.sh` or the `run.sh` but I can't for the life of me find it

EDIT I think it's maybe is a post install command `svc.sh version`

Is it possible to get the version without having to install the runner first?

**Comment by TingluoHuang** [comment, marked as answer]
```
ting@htl-mac _layout % ./run.sh --version
2.283.1
ting@htl-mac _layout % ./config.sh --version
2.283.1
```

**Reply by toast-gear** [reply]
cheers pal
